torchvision/prototype/transforms/_type_conversion.py

```diff
@@ -33,3 +37,31 @@ def extra_repr(self) -> str:
             return ""
 
         return f"num_categories={self.num_categories}"
+
+
+class ToTensor(Transform):
+    def _transform(self, input: Any, params: Dict[str, Any]) -> Any:
+        if isinstance(input, (PIL.Image.Image, np.ndarray)):
+            return F.to_tensor(input)
+        else:
+            return input
+
+
+class PILToTensor(Transform):
```

Comment:
`PILToTensor`, `ImageToPIL` names are unclear.

`PILToTensor` returns `features.Image` and `Image` in `ImageToPIL` make think of `features.Image` but apparently it is more like `ToPIL`

Reply:
Not sure what you are saying here. Do you want to change this in the future? Because the current implementation in this PR is a 1-to-1 port of what we have right now in the stable API. This is not what I'm suggesting.

Reply:
I'm saying that current names are unclear. `ToImageTensor` and `ToImagePIL` sounds much better.